nemo/collections/nlp/data/language_modeling/megatron/t5_speechlm_dataset.py

```diff
@@ -124,6 +124,8 @@ def __init__(
         spec_aug_time_masks = 2,
         cross_attention_epsilon: Optional[float] = 0.0,
         lm_vocab_size: Optional[int] = None,
+        num_speech_codebooks: Optional[int] = 8,
+        codebook_fps: Optional[int] = 76,
```

Comment:
We should default this to 75 and just do codebook_fps+1 in the class itself

Reply:
Sure, let me make that change.

Reply:
Updated